build.sh

```diff
@@ -313,20 +313,16 @@ function cleanCache {
     local dirs=("${sourcePrefix}" "${cacheDir}")
 
     local files=(
-        "${thirdpartyPath}/absl_v"*".download.success"
-        "${thirdpartyPath}/antlr_v"*".download.success"
-        "${thirdpartyPath}/grpc_v"*".download.success"
-        "${thirdpartyPath}/openBlas_v"*".download.success"
-        "${thirdpartyPath}/arrow_v"*".download.success"
+        "${thirdpartyFlags}/"*".download."*".success"
```

Comment:
Can this be included in "dirs" now?

Reply:
No, this we only want to reset the downloaded files in clearCache() if I understood it correctly. If we delete the dir, all dependencies have to be built again.

Reply:
Ah yes, sorry my bad. But I found another little glitch to nitpick about :D The install.success files in cleanDeps() still need this modification iirc. I'll fix that one when merging tomorrow morning.